aspnet/fundamentals/localization.rst

```diff
@@ -0,0 +1,337 @@
+
+::
+
+	// The QueryStringRequestCultureProvider will not locate a valid culture
+	// based on this URI as the correct locale is es-ES (case sensitive).
+	http://<yourApp>/?ui-culture=es-es
+
+One last note about the ``QueryStringRequestLocalizationProvider`` is that if either culture or ui-culture is not valid, the result is null - meaning that a culture was not located. The next provider in the list (if there is one) is then given the opportunity to determine the culture.
+
+Using the CookieRequestCultureProvider
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``CookieRequestCultureProvider`` looks for the existence of a cookie whose name is represented by ``CookieRequestCultureProvider.CookieName``. When you instantiate a ``CookieRequestCultureProvider`` object, the ``CookieRequestCultureProvider.CookieName`` value is set to the value of ``CookieRequestCultureProvider.DefaultCookieName``. The following example illustrates an example of allowing the user to create (and clear) the cookie.
+
+.. code-block:: c#
+	:emphasize-lines: 6-9,17-28
+
+	public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
+	{
+		// Add the platform handler to the request pipeline.
+		app.UseIISPlatformHandler();
+
+		var options = new RequestLocalizationOptions();
+		options.RequestCultureProviders.Clear();
+		options.RequestCultureProviders.Add(new CookieRequestCultureProvider());
+		app.UseRequestLocalization(options);
+
+		app.Run(async (context) =>
+		{
+			await context.Response.WriteAsync(
+	$@"<!doctype html>
+	<html>
+	<head>
+	<script>
+	function createCookie() {{
+		var cookieValue = '{CookieRequestCultureProvider.DefaultCookieName}=c=es-ES|uic=es-ES';
+		document.cookie = cookieValue;
+		window.location = window.location.href.split('?')[0];
+	}}
+
+	function clearCookie() {{
+		document.cookie='{CookieRequestCultureProvider.DefaultCookieName}=""""';
+		window.location = window.location.href.split('?')[0];
+	}}
+	</script>
+	</head>
+	<body>");
+
+		await context.Response.WriteAsync($"<h1>{SR["Hello"]}</h1>");
+		await context.Response.WriteAsync($"<input type=\"button\" value=\"Create es-ES culture cookie\" onclick='createCookie();' /> ");
+		await context.Response.WriteAsync($"<input type=\"button\" value=\"Clear es-ES culture cookie\" onclick='clearCookie();' /> ");
+		await context.Response.WriteAsync(
+	@"</body>
+	</html>");
+		});
+	}
+
+As you can see from the code above, the culture is prefixed with "c=" and the UI culture is prefixed with "uic=". The two values are then delimited with a vertical bar.
+
+In order to specify a different cookie name, you simply set the ``CookieRequestCultureProvider.CookieName`` when you instantiate the ``CookieRequestCultureProvider`` object. You then specify the ``CookieRequestCultureProvider.CookieName`` when you create or clear the cookie. The following code snippet illustrates doing this.
+
+.. code-block:: c#
```

Comment:
I would remove this example as well. This is an advanced scenario and folks can figure out how to set the CookieName property.